fuel-rxjava/src/main/kotlin/com/github/kittinunf/fuel/rx/RxFuel.kt

```diff
@@ -11,25 +11,25 @@ import com.github.kittinunf.result.Result
 import io.reactivex.Single
 import java.nio.charset.Charset
 
-fun Request.rx_response() = rx_response(ByteArrayDeserializer())
+fun Request.toRxResponse() = toRxResponse(ByteArrayDeserializer())
 
-fun Request.rx_responseString(charset: Charset = Charsets.UTF_8) = rx_response(StringDeserializer(charset))
+fun Request.toRxResponseString(charset: Charset = Charsets.UTF_8) = toRxResponse(StringDeserializer(charset))
 
-fun <T : Any> Request.rx_responseObject(deserializable: Deserializable<T>) = rx_response(deserializable)
+fun <T : Any> Request.toRxResponseObject(deserializable: Deserializable<T>) = toRxResponse(deserializable)
```

Comment:
Hmm I thought we have changed to `rxResponseObject` already in master which is good. Why do we change again here with prefixing it with `to`?